openbb_terminal/dashboards/dashboards_controller.py

```diff
@@ -56,6 +57,7 @@ def print_help(self):
         mt.add_cmd("chains")
         mt.add_cmd("shortdata")
         mt.add_cmd("crypto")
+        mt.add_cmd("futures")
```

Comment:
You forgot to add this to the language files, it currently doesn't have any description.